AdyenCard/Components/Card/CardComponentConfiguration.swift

```diff
@@ -21,6 +21,19 @@ extension CardComponent {
         case none
     }
 
+    /// The mode of imput field on Component UI
+    public enum FieldMode {
```

Comment:
maybe a better name would be `FieldVisibilityMode`, since there can be other modes

Reply:
👍